cea/interfaces/dashboard/inputs/static/js/map.js

```diff
@@ -0,0 +1,274 @@
+const {DeckGL, GeoJsonLayer} = deck;
+var deckgl;
+var jsonStore = {};
+// Make sure the layers maintain their order
+var layers = [new GeoJsonLayer({id:'streets'}), new GeoJsonLayer({id:'dh_networks'}),
+    new GeoJsonLayer({id:'dc_networks'}), new GeoJsonLayer({id:'zone'}), new GeoJsonLayer({id:'district'})];
+var extruded = false;
+var currentViewState;
+
+const jsonUrls = {
+    'zone': 'http://localhost:5050/inputs/geojson/zone',
+    'district': 'http://localhost:5050/inputs/geojson/district',
+    'streets': 'http://localhost:5050/inputs/geojson/others/streets',
+    'dh_networks': 'http://localhost:5050/inputs/geojson/networks/DH',
+    'dc_networks': 'http://localhost:5050/inputs/geojson/networks/DC'
+};
+
+$(document).ready(function() {
+    // Get zone file first. Will not create map if zone file does not exist
+    $.getJSON(jsonUrls['zone'], function (json) {
+        console.log(json);
+        jsonStore['zone'] = json;
+        createLayer('zone');
+        $('#zone-cb').prop('hidden', false);
+        currentViewState = {latitude: (jsonStore['zone'].bbox[1] + jsonStore['zone'].bbox[3]) / 2,
+            longitude: (jsonStore['zone'].bbox[0] + jsonStore['zone'].bbox[2]) / 2,
+            zoom: 16, bearing: 0, pitch: 0};
+
+        deckgl = new DeckGL({
+            container: 'mapid',
+            mapboxApiAccessToken: 'pk.eyJ1IjoidWJlcmRhdGEiLCJhIjoiY2pudzRtaWloMDAzcTN2bzN1aXdxZHB5bSJ9.2bkj3IiRC8wj3jLThvDGdA',
```

Comment:
i'm not sure we're suppost to publish api access tokens in the repository. could you please double-check this?

Reply:
@reyery regarding the api access token. did you look into this?

Reply:
@daren-thomas hmm I'm not too sure how to handle this also. Technically the key is a "public" key according to mapbox. We could add the api key to a file and `gitignore` it? I'm still looking into whether we can run it without a api key, since it only provides us with the map styles.